prototype_source/pt2e_quant_ptq_static.rst

```diff
@@ -352,10 +355,13 @@ Here is how you can use ``torch.export`` to export the model:
 
 .. code-block:: python
 
-    import torch._dynamo as torchdynamo
+    from torch._export import capture_pre_autograd_graph
 
     example_inputs = (torch.rand(2, 3, 224, 224),)
-    exported_model, _ = torchdynamo.export(model_to_quantize, *example_inputs, aten_graph=True, tracing_mode="symbolic")
+    exported_model, _ = capture_pre_autograd_graph(model_to_quantize, *example_inputs)
```

Comment:
Do we need to add the dynamic input constraints here to represent "symbolic"?

Reply:
Also, there's an extra `_` here; `capture_pre_autograd_graph` returns a single item, not a tuple

Reply:
we don't need "symbolic", but people can use constraints to express this, I can add a comment for dynamic_dim